addons/folo/jaxrs/src/main/java/org/commonjava/indy/folo/bind/jaxrs/FoloContentAccessResource.java

```diff
@@ -59,7 +59,7 @@
  */
 @Api( value = "FOLO Tracked Content Access and Storage",
       description = "Tracks retrieval and management of file/artifact content." )
-@Path( "/api/folo/track/{id}/{packageType}/{type: (hosted|group|remote)}/{name}" )
+@Path( "/api/folo/track/{id}/{packageType: (maven|generic)}/{type: (hosted|group|remote)}/{name}" )
```

Comment:
We might want to trim this back to just Maven. Generic content would *probably* work here, but I guess the NPM case proves what happens when we assume. :smile: We designed this interface for Maven content, so maybe we should limit it here...

Reply:
Ok. So maybe we should also change this class to more "maven" specific, like class name. :)